src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -854,7 +829,14 @@ class AddItemsPage extends Component {
     const lineItems = _.filter(formValues.lineItems, item => !_.isEmpty(item));
 
     if (_.some(lineItems, item => !item.quantityRequested || item.quantityRequested === '0')) {
-      this.confirmSave(() => this.saveItems(lineItems));
+      this.confirmSave(() => {
+        this.saveItems(lineItems);
+        const filteredLineItems = lineItems.filter(item => item.quantityRequested > 0);
```

Comment:
you could also do:
```suggestion
        const totalCount = lineItems.reduce((acc, item) => (item.quantityRequested > 0 ? acc + 1 : acc), 0);
```

Reply:
Under this line I am using `FilteredLineItems`, so why should I change this to totalCount?

```        
this.setState({
      values: { lineItems: filteredLineItems },
      totalCount: filteredLineItems.length,
  });
```

Reply:
oh, I haven't seen, that you are using `filteredLineItems` anywhere else other than `totalCount`. I thought you are doing this `.filter` only to get the `totalCount` of it.
Then my preposition has no sense, since it adds additional, unnecessary operation